src/features/attachments/utils/sorting.ts

```diff
@@ -15,12 +15,12 @@ export function mergeAndSort(...args: IAttachments[]) {
   }
 
   // Sort all attachments by name
-  for (const nodeId in Object.keys(result)) {
+  Object.keys(result).forEach((nodeId) => {
```

Comment:
This code produced a bug with the `attachments-in-groups.ts` cypress test. It iterated over the indices of the array instead of the keys. i.e ` [0,1,2,3]` vs `["key-1", "key2"]`. 

I guess it appeared because the state previousy was stored in a sorted order, and this now might not be the case. I am unsure why this has changed, but since the result always is sorted when we extract it, I don't see an   issue here.